cfme/tests/intelligence/reports/test_generate_report.py

```diff
@@ -53,8 +53,4 @@ def test_reports_generate_report(request, path, appliance):
         subtype=path[1],
         menu_name=path[2]
     ).queue(wait_for_finish=True)
```

Comment:
as you are adding some changes to this test case then please update polarion tags like add steps under polarion info. and tags if possible.